conf/deployment/aws/gp2_storage_class.yaml

```diff
@@ -0,0 +1,4 @@
+---
+# This config is for AWS deployment with gp2 CSI storage class provisioning
+DEPLOYMENT:
+  gp2: True
```

Comment:
gp2 is the default SC so not sure we need this conf file

Reply:
But from 4.10 gp3-csi will be the default SC - that's why I added this gp2 YAML file.

Reply:
oh didn't know that. So please ignore